### PR TITLE
CASMINST-4271 streamline install improvements in docs for 1.2

### PR DESCRIPTION
**Pull request by alexanderkingh**
## Summary and Scope

Summary: This PR includes a number of fresh install documentation changes. The changes vary from simple spelling corrections, to removal of superfluous filler content, better clarity on wording, as well as a great deal of syhronization of styling, single-voice write, and format across the docs and a number of other changes. Most of the style changes come from open standards styling guides as well as align with other NIST, IETF, IEEE, as well as align with other internal document styling. I will be also creating a boiler plate template for our documentation pages that we can use as a template for adding/removing sections and pages over time so that we have more consistency in the future.

Some but not all of the changes include:
1. We use a mix UPPERCASE and LOWERCASE for notes, warning and this synchronize that. The mix was an inconsistency and can be distracting.
2. Remove the use of "Air quotes" in a few places that of debatably not the best form and very distracting to the reader. Replaced the air quotes with bold *emphasis*.
3. A number of sections had some nesting and indentation issues which made the doc hard to follow which were corrected.
4. Removal of general "it", "that" and other generic wording used to refer to a previous command or step.
5. Some pages and sections from some authors included a style that was not used in other pages, this PR removes one-use styles and superfluous lines to synchronize the style across pages.
6. Addition of a table of contents Topics section for pages missing them to make them more navigable.
7. More and additional improvements not listed above and more coming

There are some additional edits to other pages of the install being added to this branch or to another branch this week.

Backwards compatible doesn't apply to the docs.

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [CASMINST-4271](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4271)
* Change will also be needed in `<insert branch name here>`
* Future work required by [issue id](issue link)
* Documentation changes required in [issue id](issue link)
* Merge with/before/after `<insert PR URL here>`

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `<development system>`
  * Local development environment
  * Virtual Shasta

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)?
- Were continuous integration tests run? If not, why?
- Was upgrade tested? If not, why?
- Was downgrade tested? If not, why?
- Were new tests (or test issues/Jiras) created for this change?

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: install/bootstrap_livecd_remote_iso.md [hunk 1]
```diff
@@ -122,7 +122,8 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
    ```
 
 1. <a name="set-up-site-link"></a>Set up the site-link, enabling SSH to work. You can reconnect with SSH after this step.
-   > **`NOTICE REGARDING DHCP`** If your site's network authority or network administrator has already provisioned an IPv4 address for your master node(s) external NIC(s), **then skip this step**.
+   > **`Note:`** If your site's network authority or network administrator has already provisioned a DHCP given IPv4 address for your master node(s) external NIC(s), **then skip this step**.
```
Comment (nrockershousen) [inline install/bootstrap_livecd_remote_iso.md:125]: I see all of the notes and warnings in this PR are both bolded and surrounded in backticks. In all of the /operations documentation (and in other product stream documentation), these are only bolded. 

I think we should keep all of these as bolded only to be consistent.

Comment (nrockershousen) [inline install/bootstrap_livecd_remote_iso.md:125]: I see all of the notes and warnings in this PR are both bolded and surrounded in backticks. In all of the /operations documentation (and in other product stream documentation), these are only bolded. 

I think we should keep all of these as bolded only to be consistent.

File: install/bootstrap_livecd_remote_iso.md [hunk 2]
```diff
@@ -374,20 +374,20 @@ Some files are needed for generating the configuration payload. See these topics
 
    1. Generate the system configuration
 
-      > **`NOTE`** for those more familiar with a CSM Install, this step may be skipped entirely by simple invoking pit-intas detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.
+      > **`Note:`** for those more familiar with a CSM Install, this step may be skipped entirely by simple invoking pit-intas detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.
```
Comment (nrockershousen) [inline install/bootstrap_livecd_remote_iso.md:377]:
```suggestion
      > **`Note:`** For those more familiar with a CSM install, this step may be skipped entirely by simple invoking `pit-intas` detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.
```

Comment (nrockershousen) [inline install/bootstrap_livecd_remote_iso.md:377]:
```suggestion
      > **`Note:`** For those more familiar with a CSM install, this step may be skipped entirely by simple invoking `pit-intas` detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.
```
```suggestion
      > **`Note:`** for those more familiar with a CSM Install, this step may be skipped entirely by simple invoking pit-intas detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.
```

File: install/bootstrap_livecd_remote_iso.md [hunk 3]
```diff
@@ -417,21 +417,21 @@ Some files are needed for generating the configuration payload. See these topics
...
 
       ```bash
       pit:/var/www/ephemeral/prep/# ls -1
       ```
 
-   Expected output looks similar to the following:
+       1. Expected output looks similar to the following:
```
Comment (nrockershousen) [inline install/bootstrap_livecd_remote_iso.md:426]: This shouldn't be a separate sub-step as it is just an example of output and the user is not doing an action. It should just be indented under the step above.

Comment (nrockershousen) [inline install/bootstrap_livecd_remote_iso.md:426]: This shouldn't be a separate sub-step as it is just an example of output and the user is not doing an action. It should just be indented under the step above.

File: install/bootstrap_livecd_remote_iso.md [hunk 4]
```diff
@@ -440,7 +440,7 @@ Some files are needed for generating the configuration payload. See these topics
       ```
 
    1. Generate the system config:
-      > **`NOTE`** the provided command below is an **example only**, run `csi config init --help` to print a full list of parameters that must be set. These will vary significatnly depending on ones system and site configuration.
+      > **`Note:`** the provided command below is an **example only**, run `csi config init --help` to print a full list of parameters that must be set. These will vary significatnly depending on ones system and site configuration.
```
Comment (nrockershousen) [inline install/bootstrap_livecd_remote_iso.md:443]:
```suggestion
      > **`Note:`** The following command is an **example only**, run `csi config init --help` to print a full list of parameters that must be set. These will vary significantly depending on the system and site configuration in use.
```

Comment (nrockershousen) [inline install/bootstrap_livecd_remote_iso.md:443]:
```suggestion
      > **`Note:`** The following command is an **example only**, run `csi config init --help` to print a full list of parameters that must be set. These will vary significantly depending on the system and site configuration in use.
```
```suggestion
      > **`Note:`** the provided command below is an **example only**, run `csi config init --help` to print a full list of parameters that must be set. These will vary significatnly depending on ones system and site configuration.
```

File: install/bootstrap_livecd_remote_iso.md [hunk 5]
```diff
@@ -519,28 +519,28 @@ Some files are needed for generating the configuration payload. See these topics
...
+1. Follow the procedures to [Prepare Site Init](prepare_site_init.md) directory for your system.
 
-Finally, cleanup the shim:
- ```bash
- pit# cd ~
- # this uses `rmdir` to safely remove the directory, preventing accidental removal if one does not notice a `umount` command failure.
- pit# umount -v /mnt/pitdata/
- pit# rmdir -v /mnt/pitdata
- ```
+1. Finally, cleanup the shim:
```
Comment (nrockershousen) [inline install/bootstrap_livecd_remote_iso.md:531]:
```suggestion
1. Clean-up the shim:
```

Comment (nrockershousen) [inline install/bootstrap_livecd_remote_iso.md:531]:
```suggestion
1. Clean-up the shim:
```

File: install/prepare_configuration_payload.md [hunk 6]
```diff
@@ -6,9 +6,9 @@ can be passed to the `csi` (Cray Site Init) program during the CSM installation
...
 
 How the configuration payload is prepared depends on whether this is a first time installation of CSM
-software on this system or the CSM software is being reinstalled. The reinstall scenario has the
+software on this system or the CSM software is being reinstalled. The *reinstall scenario* has the
 advantage of being able to use the configuration payload from the previous CSM installation
-and an extra configuration file which that installation generated.
+and an additional configuration file which that installation generated. The *first time* install scenario requires using the csi config tool to pass arguments to csi as well as the creation of a number of [Configuration Payload Files}(#configuration-payload-files).
```
Comment (nrockershousen) [inline install/prepare_configuration_payload.md:11]: Second instance of CSI should be capitalized.

Comment (nrockershousen) [inline install/prepare_configuration_payload.md:11]: Second instance of CSI should be capitalized.

File: install/prepare_management_nodes.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 # Prepare Management Nodes
 
-The procedures described on this page are being done before any node is booted with the Cray Pre-Install Toolkit. When the PIT node is referenced during these procedures, it means the node that will be be booted as the PIT node.
+The procedures described on this page must be completed before any node is booted with the Cray Pre-Install Toolkit which is performed in a later document. When the PIT node is referenced during these procedures, it means the node that will be booted as the PIT node.
```
Comment (nrockershousen) [inline install/prepare_management_nodes.md:3]:
```suggestion
The procedures described on this page must be completed before any node is booted with the Cray Pre-Install Toolkit (PIT), which is performed in a later document. When the PIT node is referenced during these procedures, it means the node that will be booted as the PIT node.
```

Comment (nrockershousen) [inline install/prepare_management_nodes.md:3]:
```suggestion
The procedures described on this page must be completed before any node is booted with the Cray Pre-Install Toolkit (PIT), which is performed in a later document. When the PIT node is referenced during these procedures, it means the node that will be booted as the PIT node.
```

File: install/bootstrap_livecd_remote_iso.md (hunk 1 above)
Comment (alexanderkingh) [reply install/bootstrap_livecd_remote_iso.md:125]: Thats an interesting and valid observation. These install docs however had the Note and Warnings using this existing style already. At this time we're trying to limit from making too many style, format and content changes at once that might confuse readers. For now, we're going to move ahead with this style. We are adding a linter to the docs and will review this proposed change to remove the back tics and leave just the bold at a later time.

Comment (alexanderkingh) [reply install/bootstrap_livecd_remote_iso.md:125]: Thats an interesting and valid observation. These install docs however had the Note and Warnings using this existing style already. At this time we're trying to limit from making too many style, format and content changes at once that might confuse readers. For now, we're going to move ahead with this style. We are adding a linter to the docs and will review this proposed change to remove the back tics and leave just the bold at a later time.

File: install/prepare_configuration_payload.md (hunk 6 above)
Comment (alexanderkingh) [reply install/prepare_configuration_payload.md]: Agreed, made the recommended change.

Comment (alexanderkingh) [reply install/prepare_configuration_payload.md]: Agreed, made the recommended change.

**Comment by alexanderkingh** [comment]
I have made or committed the majority of the recommended changes and resolved the conversations.

File: install/collect_mac_addresses_for_ncns.md [hunk 8]
```diff
@@ -130,11 +130,11 @@ making a backup of them, in case they need to be examined at a later time.
 
 1. Check that IP addresses are set for each interface and investigate any failures.
 
-  > Note that bond0.can0 is optional in CSM 1.2+
+    > **`Note:`** Note that bond0.can0 is optional in CSM 1.2+
```
Comment (nrockershousen) [inline install/collect_mac_addresses_for_ncns.md:133]: It seems redundant to use note twice here. Also, a period is needed at the end of the line. 

Suggested change:

> **`Note:`** bond0.can0 is optional in CSM 1.2+ releases.

File: install/collect_mac_addresses_for_ncns.md [hunk 9]
```diff
@@ -316,11 +316,11 @@ making a backup of them, in case they need to be examined at a later time.
...
```
Comment (nrockershousen) [inline install/collect_mac_addresses_for_ncns.md:319]: same comment as above

File: install/index.md [hunk 10]
```diff
@@ -33,7 +38,7 @@ Once the CSM installation has completed, other product streams for the HPE Cray
 
 The topics in this chapter need to be done as part of an ordered procedure so are shown here with numbered topics.
 
-**Note**: If problems are encountered during the installation, some topics have their own troubleshooting
+**Note**: If problems are encountered during the installation, some topics have their own [troubleshooting sections found in the operations index](../operations/index.md)
```
Comment (nrockershousen) [inline install/index.md:41]: - This line is missing a period
- Could add backticks to **`Note`** to be consistent with the other changes

**Review by nrockershousen — APPROVED** [review]

File: install/collect_mac_addresses_for_ncns.md (hunk 8 above)
Comment (alexanderkingh) [reply install/collect_mac_addresses_for_ncns.md]: Corrected

File: install/collect_mac_addresses_for_ncns.md (hunk 9 above)
Comment (alexanderkingh) [reply install/collect_mac_addresses_for_ncns.md]: Corrected

File: install/index.md (hunk 10 above)
Comment (alexanderkingh) [reply install/index.md]: Corrected

**Review by jasonrouault — APPROVED** [review]
